From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Wed, 29 Jul 2026 02:35:32 +0200
Subject: media: ov5640: Sort the SDE defines

The bit and unity value defines ended up interleaved with the register
defines as the SDE controls were fixed one by one. Group the bits with
the register they belong to.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/media/i2c/ov5640.c | 11 +++++-----
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/drivers/media/i2c/ov5640.c b/drivers/media/i2c/ov5640.c
index 111111111111..222222222222 100644
--- a/drivers/media/i2c/ov5640.c
+++ b/drivers/media/i2c/ov5640.c
@@ -123,6 +123,8 @@
 #define OV5640_REG_PRE_ISP_TEST_SET1	0x503d
 #define OV5640_REG_SDE_CTRL0		0x5580
 #define OV5640_SDE_CTRL0_HUE_EN		BIT(0)
+#define OV5640_SDE_CTRL0_SAT_EN		BIT(1)
+#define OV5640_SDE_CTRL0_CONTRAST_EN	BIT(2)
 #define OV5640_REG_SDE_CTRL1		0x5581
 #define OV5640_REG_SDE_CTRL2		0x5582
 #define OV5640_REG_SDE_CTRL3		0x5583
@@ -130,10 +132,6 @@
 #define OV5640_REG_SDE_CTRL5		0x5585
 #define OV5640_REG_SDE_CTRL6		0x5586
 #define OV5640_REG_SDE_CTRL7		0x5587
-#define OV5640_SDE_CTRL0_SAT_EN		BIT(1)
-#define OV5640_SDE_CTRL0_CONTRAST_EN	BIT(2)
-/* UV adjust gain, 0x40 is unity */
-#define OV5640_SATURATION_UNITY		0x40
 #define OV5640_REG_SDE_CTRL8		0x5588
 /*
  * Hue is a rotation of the U/V plane:
@@ -153,8 +151,9 @@
 #define OV5640_SDE_CTRL8_HUE_SIGN_MASK	(OV5640_SDE_CTRL8_HUE_SIN_POS | \
 					 OV5640_SDE_CTRL8_HUE_SIN_NEG | \
 					 OV5640_SDE_CTRL8_HUE_COS_NEG)
-/* Y gain for contrast, 0x20 is unity */
-#define OV5640_CONTRAST_UNITY		0x20
+/* Neutral settings of the SDE controls */
+#define OV5640_SATURATION_UNITY		0x40	/* UV adjust gain */
+#define OV5640_CONTRAST_UNITY		0x20	/* Y gain */
 #define OV5640_REG_AVG_READOUT		0x56a1
 
 /* autofocus registers */
-- 
Armbian

